spring-integration-jdbc/build.gradle

```diff
@@ -45,7 +47,3 @@ task generateSql {
 
 // tie schema generation to the build lifecycle
 compileJava.dependsOn generateSql
-
-task cleanSql (type: Delete) {
```

Comment:
Pushed. See commit comment.
I'd be glad to hear from you that it works properly not only from my side.
I mean this to check should be enough:
```
./gradlew clean :spring-integration-jdbc:testAll
```